test/integration/02-user-wallet/02-receive-lightning.spec.ts

```diff
@@ -181,6 +183,34 @@ describe("UserWallet - Lightning", () => {
     if (imbalance instanceof Error) throw imbalance
 
     expect(Number(imbalance.amount)).toBe(sats)
+
+    // Check ledger transaction metadata for BTC 'LedgerTransactionType.Invoice'
+    // ===
+    const usdPaymentAmount = await usdFromBtcMidPriceFn({
+      amount: BigInt(sats),
+      currency: WalletCurrency.Btc,
+    })
+    if (usdPaymentAmount instanceof Error) throw usdPaymentAmount
+    const centsAmount = Number(usdPaymentAmount.amount)
+
+    const expectedFields = {
```

Comment:
should not check display currency too? if so, please check all ledger `expectedFields` validations

Reply:
Done: https://github.com/GaloyMoney/galoy/pull/2157/commits/d3afbb66f198a2e3cb8143fa4d8810153d63401b